glide.lock

```diff
@@ -194,24 +190,32 @@ imports:
 - name: github.com/elazarl/go-bindata-assetfs
   version: 30f82fa23fd844bd5bb1e5f216db87fd77b5eb43
 - name: github.com/emicklei/go-restful
-  version: 892402ba11a2e2fd5e1295dd633481f27365f14d
+  version: 89ef8af493ab468a45a42bb0d89a06fccdd2fb22
   subpackages:
   - log
   - swagger
 - name: github.com/fatih/color
   version: 9131ab34cf20d2f6d83fdc67168a5430d1c7dc23
 - name: github.com/gambol99/go-marathon
-  version: 15ea23e360abb8b25071e677aed344f31838e403
```

Comment:
It's not only for `go-marathon`. It's a side affect of the `glide get`.
You must use a workaround:
1. Revert the glide manifest and lock files.
1. Use the [github.com/multiplay/glide-pin](https://github.com/multiplay/glide-pin) plugin to pin all versions in the manifest file to the ones given in the lock file.
1. Add your dependencies `glide get`
1. Run `glide update -v` to update the lock file.
1. Undo the pinning.
1. Add your dependencies in `glide.yml`
1. Use [glide-hash](https://github.com/mattfarina/glide-hash) to get the manifest file hash and put it in the lock file to please `validate-glide`.